### ISSUE TITLE
MS Word (form templates / Macros)

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7087]
Dear NVDA users and developers,

I’ve been using NVDA for more than 5 years. I used Jaws, Window Eyes, Cobra and also Dolphin Super Nova. Jaws and NVDA seems to be the most reliant screen readers. Other screen readers worked really slow even on gaming pcs according to my tests.
 
I’ve discovered some issues with NVDA which I would like to make you familiar with. It would help a lot if you could solve them with the next updates. My results come also from lots of other blind and visually impaired people I discussed with.

I can reproduce these issues in MS Word 2010 with NVDA 2017.4 and 2017.1.

•	In forms where there are Macros implemented, NVDA doesn’t read aloud the name of them. Jaws does
•	Sometimes there are fields which indicate what is required to type in a certain area. Most of them are yellow. This indication fields are not recognized by NVDA at all. When I move with the cursor on such a field, NVDA doesn’t read anything. Jaws says the indication message

Best regards and many thanks to all developers which are part of this community and make possible such an useful project to be maintained and updated
Adriani

**Comment by Adriani90** [comment]
I have slightly edited this issue.

**Comment by Adriani90** [comment]
Closing, this got much better now in NVDA alpha-53423,57f2e9dc (2026.1.0.53423)
and Microsoft 365 MSO (Version 2510 Build 16.0.19328.20190) 64 Bit.
